readhomer_atlas/tests/constants.py

```diff
@@ -12,13 +12,115 @@
 
 
 VERSION_DATA = {
+    "path": "data/library/tlg0012/tlg001/tlg0012.tlg001.perseus-grc2.txt",
     "urn": "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:",
-    "content_path": "tlg0012.tlg001.perseus-grc2.txt",
-    "metadata": {
-        "work_title": "Iliad",
-        "work_urn": "urn:cts:greekLit:tlg0012.tlg001:",
-        "type": "edition",
-        "first_passage_urn": "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1-1.7",
-        "citation_scheme": ["rank_1", "rank_2"],
+    "nodeKind": "version",
+    "versionKind": "edition",
+    "firstPassageUrn": "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1-1.7",
+    "citationScheme": ["book", "line"],
+    "title": [{"lang": "eng", "value": "Iliad, Homeri Opera"}],
+    "description": [
+        {
+            "lang": "eng",
+            "value": "Homer, creator; Monro, D. B. (David Binning), 1836-1905, creator; Monro, D. B. (David Binning), 1836-1905, editor; Allen, Thomas W. (Thomas William), b. 1862, editor",
+        }
+    ],
+}
+
+VERSION_METADATA = {
```

Comment:
What is the protocol now for `camelCase` vs. `snake_case`? I see some instances of both here in the fixtures, is there a semantic reason? I previously thought we were keeping everything `snake_case` ATLAS-side and then relying on `camelize` when transmitting anything to the frontend...

Reply:
Good attention to detail; I think as with `upTo` above, where we can enforce consistency with `snake_case`, we should:

- If we're specifying a format for ATLAS (like `metadata.json`), we should use `snake_case`
- That allows us to depend on `snake_case` internally within our Python code
- When returning a GraphQL response, we'll leverage the `camelize` helper to present structured data in `camelCase`